docker/Dockerfile

```diff
@@ -6,7 +6,7 @@ ENV PYTHONIOENCODING UTF-8
 ARG DEBIAN_FRONTEND=noninteractive
 
 # Pypy3 is installed from a package manager because it takes so long to build.
-RUN apt-get update && apt-get install -y build-essential \
+RUN apt-get update && apt-get install --no-install-recommends -y build-essential \
```

Comment:
Are you sure this change was needed?

Reply:
bit sure but willl cross check and remove if these are the root cause

Reply:
@auvipy yeah this line breaks the build.
I just tried locally, with/without to verify.

Reply:
Ok so removing it passes a certain stage it didn't with it, but the build just failed locally.
I'll try to see what could have caused it

Reply:
https://github.com/celery/celery/pull/8614/commits/6206a350dce8abfb99541e3a7956be261148a8a9 I added the package for fixing certificate error